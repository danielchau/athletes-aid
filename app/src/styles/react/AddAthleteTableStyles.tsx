import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

/**
 * React styles for the Add Athletes Table.
 */
export const addAthleteTableStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            padding: "8px",
            height: "100%",
            display: "flex",
            justifyContent: "center"
        },
        tableContainer: {
            width: "100%",
            height: "100%",
            padding: "8px",
            backgroundColor: "rgba(0, 33, 69, 0.05)",
            overflow: "scroll"
        },
        table: {
            width: "100%"
        },
        tableBodyContainer: {
            width: "2000px",
            overflow: "auto",
            height: "100%"
        },
        tableBody: {
            tableLayout: "fixed"
        },
        tableRow: {
            cursor: "pointer"
        },
        tooltip: {
            backgroundColor: "#fff",
            color: "#4f4f4f",
            boxShadow: theme.shadows[1]
        }
    })
);
