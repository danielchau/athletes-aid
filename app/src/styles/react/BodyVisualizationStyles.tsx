import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

/**
 * React styles for the Body Visualization.
 */
export const bodyVisualizationStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            height: "100%"
        },
        imageContainer: {
            position: "relative",
            alignSelf: "center"
        },
        image: {
            width: "100%"
        },
        dot: {
            height: "15px",
            width: "15px",
            position: "absolute",
            borderRadius: "15px"
        },
        legendContainer: {
            borderLeft: "#dedede solid 1px"
        },
        legendItem: {
            display: "flex",
            color: "rgba(0, 0, 0, 0.54)",
            marginBottom: "2px"
        },
        dotLegend: {
            height: "10px",
            width: "10px",
            borderRadius: "10px",
            marginTop: "6px",
            marginRight: "4px"
        }
    })
);
