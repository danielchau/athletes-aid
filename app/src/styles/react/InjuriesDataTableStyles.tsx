import { makeStyles, Theme, createStyles, lighten } from "@material-ui/core/styles";

/**
 * React styles for the Injury Data Table.
 */
export const injuriesDataTableStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            marginTop: theme.spacing(3)
        },
        paper: {
            width: "100%",
            marginBottom: theme.spacing(2),
            backgroundColor: "rgba(0, 33, 69, 0.05)"
        },
        table: {
            minWidth: 750
        },
        tableWrapper: {
            overflowX: "auto"
        },
        tableRow: {
            cursor: "pointer"
        },
        visuallyHidden: {
            border: 0,
            clip: "rect(0 0 0 0)",
            height: 1,
            margin: -1,
            overflow: "hidden",
            padding: 0,
            position: "absolute",
            top: 20,
            width: 1
        }
    })
);

/**
 * React styles for the Injury Data Table Toolbar Header.
 */
export const injuriesDataTableToolbarStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            paddingLeft: theme.spacing(2),
            paddingRight: theme.spacing(1)
        },
        highlight:
            theme.palette.type === "light"
                ? {
                      color: theme.palette.primary.main,
                      backgroundColor: lighten(theme.palette.primary.light, 0.85)
                  }
                : {
                      color: theme.palette.text.primary,
                      backgroundColor: theme.palette.secondary.dark
                  },
        title: {
            flex: "1 1 100%"
        },
        export: {
            margin: "auto",
            color: theme.palette.secondary.main,
            display: "flex"
        }
    })
);
