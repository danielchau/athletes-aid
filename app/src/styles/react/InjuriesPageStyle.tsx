import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

/**
 * React styles for the Injuries Page.
 */
export const injuriesPageStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            padding: "16px",
            backgroundColor: "#fff"
        },
        drawerOpen: {
            width: "calc(100vw - 240px)",
            transition: theme.transitions.create("width", {
                easing: theme.transitions.easing.sharp,
                duration: theme.transitions.duration.enteringScreen
            }),
            [theme.breakpoints.down("xs")]: {
                width: "100vw"
            }
        },
        drawerClosed: {
            width: "calc(100vw - 72px)",
            transition: theme.transitions.create("width", {
                easing: theme.transitions.easing.sharp,
                duration: theme.transitions.duration.leavingScreen
            }),
            [theme.breakpoints.down("xs")]: {
                width: "100vw"
            }
        },
        grid: {
            backgroundColor: "#fff"
        },
        paper: {
            padding: theme.spacing(2),
            textAlign: "center",
            color: theme.palette.text.secondary,
            backgroundColor: "rgba(0, 33, 69, 0.05)",
            height: "100%"
        },
        vizPaper: {
            padding: theme.spacing(2),
            height: "100%"
        },
        primaryStatisticContainer: {
            display: "flex",
            flexDirection: "column"
        },
        primaryStatisticValue: {
            overflow: "hidden",
            textOverflow: "ellipsis",
            fontSize: "calc((100vw - 240px)/16)"
        },
        primaryStatisticLabel: {
            paddingTop: "16px"
        },
        dateTimeContainer: {},
        textField: {
            marginLeft: theme.spacing(1),
            marginRight: theme.spacing(1),
            width: 200
        },
        dateField: {
            width: 200,
            marginRight: theme.spacing(1),
            marginTop: 0,
            marginBottom: 0
        },
        dateTimeButton: {
            marginLeft: "8px",
            marginTop: "6px"
        }
    })
);
