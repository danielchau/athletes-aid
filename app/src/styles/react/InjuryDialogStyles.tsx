import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

export const injuryDialogStyles = makeStyles((theme: Theme) =>
    createStyles({
        appBar: {
            position: "relative"
        },
        title: {
            marginLeft: theme.spacing(2),
            flex: 1
        },
        dialogTitle: {
            padding: "0"
        },
        dialogPaper: {
            height: "100%",
            maxWidth: "600px",
            width: "50%",
            minWidth: "200px",
            position: "fixed",
            bottom: "0",
            right: "0",
            margin: "0"
        },
        dialogContentContainer: {
            padding: "0"
        },
        dialogContent: {
            padding: "16px"
        },
        dialogContentPaper: {
            backgroundColor: "#fafafa",
            padding: "8px",
            marginBottom: "8px"
        },
        notePaper: {
            backgroundColor: "#ffffff",
            padding: "8px",
            marginBottom: "8px"
        },
        noteContainer: {
            display: "flex",
            flexDirection: "row"
        },
        noteIcon: {
            margin: "auto",
            marginLeft: "8px",
            marginRight: "16px"
        },
        newNoteContainer: {
            display: "flex",
            flexDirection: "row",
            width: "100%",
            justifyContent: "center",
            alignItems: "center",
            alignSelf: "center"
        },
        newNote: {
            flexGrow: 1,
            paddingRight: "8px"
        },
        sendButton: {
            color: "#F2A71E"
        },
        activeSwitchContainer: {
            display: "flex",
            flexDirection: "row",
            justifyContent: "center",
            alignContent: "center",
            alignSelf: "center",
            paddingBottom: "8px"
        }
    })
);
