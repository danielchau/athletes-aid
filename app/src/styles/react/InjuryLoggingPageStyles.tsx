import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

export const injuryLoggingPageStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            width: "100%",
            padding: "8px"
        },
        backButton: {
            marginRight: theme.spacing(1)
        },
        completedContainer: {
            height: "calc(100% - 116px)",
            textAlign: "center",
            paddingTop: "20%"
        },
        instructions: {
            marginTop: theme.spacing(1),
            marginBottom: theme.spacing(1)
        },
        paper: {
            color: theme.palette.text.secondary,
            backgroundColor: "#fafafa"
        },
        paperContent: {
            height: "100%",
            padding: "16px",
            paddingRight: "8px",
            overflow: "auto"
        },
        stepper: {
            backgroundColor: "#fafafa",
            borderRadius: "4px"
        },
        loggingContent: {
            height: "calc(100% - 168px)",
            marginBottom: "52px",
            marginTop: "8px",
            overflow: "hidden"
        },
        loggingBottomButtons: {
            position: "fixed",
            bottom: 0,
            right: 0,
            height: "52px",
            padding: "8px"
        },
        multilineInput: {
            width: "100%"
        }
    })
);
