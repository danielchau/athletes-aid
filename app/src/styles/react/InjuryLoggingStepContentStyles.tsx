import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";

export const injuryLoggingStepContentStyles = makeStyles((theme: Theme) =>
    createStyles({
        formControl: {
            minWidth: 200,
            width: "50%",
            paddingRight: "8px",
            paddingBottom: "16px"
        },
        textInput: {
            minWidth: 200,
            width: "100%",
            paddingRight: "8px",
            paddingBottom: "16px"
        },
        multilineInput: {
            width: "100%",
            paddingBottom: "16px",
            margin: "0px"
        },
        dateInput: {
            minWidth: 200,
            width: "50%",
            margin: "0px",
            paddingRight: "8px",
            paddingBottom: "16px"
        },
        checkboxInput: {
            minWidth: 200,
            width: "50%",
            margin: "0px",
            paddingRight: "8px",
            paddingBottom: "23px",
            paddingTop: "7px"
        },
        instructions: {
            textAlign: "center",
            marginBottom: theme.spacing(3),
            color: theme.palette.text.secondary,
            width: "100%"
        }
    })
);
