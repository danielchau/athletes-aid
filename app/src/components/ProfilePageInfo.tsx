import React from "react";
import { AthleteProfile, User } from "../util/types";
import { Typography, Divider, TextField } from "@material-ui/core";
import { profilePageStyles } from "../styles/react/ProfilePageStyles";
import { MuiPickersUtilsProvider, KeyboardDatePicker } from "@material-ui/pickers";
import DateFnsUtils from "@date-io/date-fns";
import { updateAthlete } from "../actions/AthleteAction";

interface ProfilePageInfo {
    currentAthlete: AthleteProfile;
    isEditing: boolean;
    setIsUpdating: any;
    currentUser: User;
}

/**
 * Renders all of the basic information for an athlete on their profile.
 * @param props
 */
export default function ProfilePageInfo(props: ProfilePageInfo) {
    const classes = profilePageStyles({});
    const [isInitialRender, setIsInitialRender] = React.useState<boolean>(true);
    const [birthdate, setBirthdate] = React.useState<string>(props.currentAthlete.birthdate);
    const [schoolYear, setSchoolYear] = React.useState<string>(
        props.currentAthlete.schoolYear.toString()
    );
    const [gender, setGender] = React.useState<string>(props.currentAthlete.gender);
    const [weight, setWeight] = React.useState<string>(props.currentAthlete.weight.toString());
    const [height, setHeight] = React.useState<string>(props.currentAthlete.height.toString());

    const [email, setEmail] = React.useState<string>(props.currentAthlete.email);
    const [cellPhone, setCellPhone] = React.useState<string>(props.currentAthlete.cellPhone);
    const [homePhone, setHomePhone] = React.useState<string>(props.currentAthlete.homePhone);
    const [healthCardNo, setHealthCardNo] = React.useState<string>(
        props.currentAthlete.healthCardNumber
    );
    const [studentNo, setStudentNo] = React.useState<string>(props.currentAthlete.studentNumber);

    const [ecName, setEcName] = React.useState<string>(props.currentAthlete.emergencyContact.name);
    const [ecEmail, setEcEmail] = React.useState<string>(
        props.currentAthlete.emergencyContact.email
    );
    const [ecPhone, setEcPhone] = React.useState<string>(
        props.currentAthlete.emergencyContact.phone
    );

    React.useEffect(() => {
        setBirthdate(props.currentAthlete.birthdate);
        setSchoolYear(props.currentAthlete.schoolYear.toString());
        setGender(props.currentAthlete.gender.toString());
        setWeight(props.currentAthlete.weight.toString());
        setEmail(props.currentAthlete.email);
        setCellPhone(props.currentAthlete.cellPhone);
        setHomePhone(props.currentAthlete.homePhone);
        setHealthCardNo(props.currentAthlete.healthCardNumber);
        setStudentNo(props.currentAthlete.studentNumber);
        setEcName(props.currentAthlete.emergencyContact.name);
        setEcEmail(props.currentAthlete.emergencyContact.email);
        setEcPhone(props.currentAthlete.emergencyContact.phone);
    }, [props.currentAthlete]);

    React.useEffect(() => {
        if (!props.isEditing && !isInitialRender) {
            updateAthlete(
                {
                    id: props.currentAthlete.id,
                    profilePicture: props.currentAthlete.profilePicture,
                    name: props.currentAthlete.name,
                    birthdate: birthdate,
                    schoolYear: Number(schoolYear),
                    gender: gender,
                    weight: Number(weight),
                    height: Number(height),
                    email: email,
                    cellPhone: cellPhone,
                    homePhone: homePhone,
                    healthCardNumber: healthCardNo,
                    studentNumber: studentNo,
                    emergencyContact: {
                        id: props.currentAthlete.emergencyContact.id,
                        name: ecName,
                        phone: ecPhone,
                        email: ecEmail
                    },
                    files: props.currentAthlete.files,
                    injuries: props.currentAthlete.injuries
                },
                props.currentUser.athleteProfile.name
            ).then(_ => {
                props.setIsUpdating(false);
            });
        }
        setIsInitialRender(false);
    }, [props.isEditing]);

    return (
        <>
            <Typography className={classes.heading} variant="h6">
                Basic Information
            </Typography>
            {[
                ["Birthdate", birthdate, setBirthdate],
                ["Year In School", schoolYear, setSchoolYear],
                ["Gender", gender, setGender],
                ["Weight", weight, setWeight],
                ["Height", height, setHeight]
            ].map(([category, value, event], i: number) => (
                <ProfileAttribute
                    key={i}
                    category={category.toString()}
                    value={value.toString()}
                    isEditing={props.isEditing}
                    onChange={event}
                />
            ))}
            <Divider light />
            <Typography className={classes.heading} variant="h6">
                Contact Information
            </Typography>
            {[
                ["Email", email, setEmail],
                ["Cell Phone", cellPhone, setCellPhone],
                ["Home Phone", homePhone, setHomePhone],
                ["Student Number", studentNo, setStudentNo],
                ["Health Card Number", healthCardNo, setHealthCardNo]
            ].map(([category, value, event], i: number) => (
                <ProfileAttribute
                    key={i}
                    category={category.toString()}
                    value={value.toString()}
                    isEditing={props.isEditing}
                    onChange={event}
                />
            ))}
            <Divider light />
            <Typography className={classes.heading} variant="h6">
                Emergency Contact Information
            </Typography>
            {[
                ["Name", ecName, setEcName],
                ["Email", ecEmail, setEcEmail],
                ["Phone", ecPhone, setEcPhone]
            ].map(([category, value, event], i: number) => (
                <ProfileAttribute
                    key={i}
                    category={category.toString()}
                    value={value.toString()}
                    isEditing={props.isEditing}
                    onChange={event}
                />
            ))}
        </>
    );
}

interface ProfileAttributeProps {
    category: string;
    value: string;
    isEditing: boolean;
    onChange: any;
}

/**
 * Renders an attribute on an athlete page.
 * Two states: Editing and not editing.
 * @param props
 */
function ProfileAttribute(props: ProfileAttributeProps) {
    const inputStyle = {
        marginTop: "-2px",
        backgroundColor: "rgba(0, 33, 69, 0.05)",
        borderRadius: "5px",
        flexGrow: 1,
        marginBottom: "0px"
    };

    return (
        <div style={{ display: "flex", paddingBottom: "4px" }}>
            <Typography variant="body1" style={{ paddingRight: "4px", fontWeight: 400 }}>
                {props.category}:
            </Typography>
            {props.isEditing ? (
                props.category == "Birthdate" ? (
                    <MuiPickersUtilsProvider utils={DateFnsUtils}>
                        <KeyboardDatePicker
                            style={inputStyle}
                            disableToolbar
                            variant="inline"
                            format="MM/dd/yyyy"
                            margin="normal"
                            id="birthdate"
                            value={new Date(props.value)}
                            onChange={(date: Date) => props.onChange(date.toDateString())}
                        />
                    </MuiPickersUtilsProvider>
                ) : (
                    <TextField
                        style={inputStyle}
                        value={props.value}
                        onChange={(event: any) => props.onChange(event.target.value)}
                    />
                )
            ) : (
                <Typography style={{ fontWeight: 300 }} variant="body1">
                    {props.value}
                </Typography>
            )}
            {props.category == "Weight" && (
                <Typography style={{ fontWeight: 300, paddingLeft: "4px" }} variant="body1">
                    kg
                </Typography>
            )}
            {props.category == "Height" && (
                <Typography style={{ fontWeight: 300, paddingLeft: "4px" }} variant="body1">
                    cm
                </Typography>
            )}
        </div>
    );
}
