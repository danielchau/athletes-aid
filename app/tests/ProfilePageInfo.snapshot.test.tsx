import * as React from 'react'
import ProfilePageInfo from '../src/components/ProfilePageInfo'
import renderer from 'react-test-renderer'

let date: Date = new Date("2019-01-01");

const mockInjuryNote = {
    createdBy: "Test createdBy",
    createdOn: date,
    content: "Test content"
}

const mockEmergencyContact = {
    id: "Test id",
    name: "Test name",
    phone: "Test phone",
    email: "Test email"
}

const mockInjury = {
    id: "Test id",
    active: true,
    createdOn: date,
    createdBy: "Test createdBy",
    teamName: "Test teamName",
    athleteName: "Test athleteName",
    injuryDate: date,
    isSportsRelated: true,
    eventType: "Test eventType",
    position: "Test position",
    sideOfBody: "Test sideOfBody",
    locationOnBody: "Test locationOnBody",
    injuryType: "Test injuryType",
    severity: 1,
    status: "Test status",
    mechanism: "Test mechanism",
    injuryDescription: "Test injuryDescription",
    otherNotes: [mockInjuryNote]
}

const mockAthlete = {
    id: "Test id",
    profilePicture: "Test profilePicture",
    name: "Test name",
    birthdate: "Test birthdate",
    schoolYear: 1,
    gender: "Test gender",
    weight: 1,
    height: 1,
    email: "Test email",
    cellPhone: "Test cellPhone",
    homePhone: "Test homePhone",
    healthCardNumber: 1,
    studentNumber: "Test studentNumber",
    emergencyContact: mockEmergencyContact,
    files: ["Test files"],
    injuries: [mockInjury]
}

const mockUser = {
    athleteProfile: {},
    permissions: {}
}

const mockTeam = {
    id: "Test id",
    name: "Test name",
    season: "Test season",
    athleteIds: ["Test athleteId1", "Test athleteId2"]
}


it('ProfilePageInfo renders correctly', () => {
  const tree = renderer.create(<ProfilePageInfo currentAthlete={mockAthlete} isEditing={false} currentUser={mockUser} />).toJSON()
  expect(tree).toMatchSnapshot()
})

/*
interface ProfilePageInfo {
    currentAthlete: AthleteProfile;
    isEditing: boolean;
    setIsUpdating: any;
    currentUser: User;
}
*/
