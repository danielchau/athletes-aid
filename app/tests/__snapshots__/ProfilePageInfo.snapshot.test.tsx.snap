// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilePageInfo renders correctly 1`] = `
Array [
  <h6
    className="MuiTypography-root makeStyles-heading-8 MuiTypography-h6"
  >
    Basic Information
  </h6>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Birthdate
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test birthdate
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Year In School
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      1
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Gender
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test gender
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Weight
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      1
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
          "paddingLeft": "4px",
        }
      }
    >
      kg
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Height
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      1
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
          "paddingLeft": "4px",
        }
      }
    >
      cm
    </p>
  </div>,
  <hr
    className="MuiDivider-root MuiDivider-light"
  />,
  <h6
    className="MuiTypography-root makeStyles-heading-8 MuiTypography-h6"
  >
    Contact Information
  </h6>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Email
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test email
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Cell Phone
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test cellPhone
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Home Phone
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test homePhone
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Student Number
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test studentNumber
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Health Card Number
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      1
    </p>
  </div>,
  <hr
    className="MuiDivider-root MuiDivider-light"
  />,
  <h6
    className="MuiTypography-root makeStyles-heading-8 MuiTypography-h6"
  >
    Emergency Contact Information
  </h6>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Name
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test name
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Email
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test email
    </p>
  </div>,
  <div
    style={
      Object {
        "display": "flex",
        "paddingBottom": "4px",
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 400,
          "paddingRight": "4px",
        }
      }
    >
      Phone
      :
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1"
      style={
        Object {
          "fontWeight": 300,
        }
      }
    >
      Test phone
    </p>
  </div>,
]
`;
